src/data/projects.json

```diff
@@ -23,7 +23,7 @@
       "web": "https://www.connext.network/"
     },
     {
-      "name": "Crosschain Governance (with Connext)",
+      "name": "Cross-chain Governance (with Connext)",
```

Comment:
I believe this is a typo because in the previous project's description we say "cross-chain" with the dash.